src/components/LoggedInLayout/LoggedInLayout.jsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { RedBackground } from '../index';
-import './LoggedInLayout.sass';
+import React from "react";
+import PropTypes from "prop-types";
+import { RedBackground } from "../index";
+import "./LoggedInLayout.sass";
```

Comment:
Did you change the formatting here manually or was it something your IDE did? Can you change it back to be consistent with everywhere else? Double quotes should be only used in HTML (like below in line 32). Please make sure your IDE is not changing things on its own.